### PR TITLE
(feat) Face Detector

### PR DESCRIPTION
**Pull request by jarulraj**
-- Added UDF and an integration test case

### CODE REVIEW

**Comment by jarulraj** [comment]
Do we need `labels` and `input format` properties for all UDFs?

**Comment by gaurav274** [comment]
No, we don't need them. In fact, we are not yet enforcing the `input_format`.

**Comment by LordDarkula** [comment]
I added defaults for `name` and `input_format` in #296

File: eva/udfs/face_detector.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        Performs predictions on input frames
+        Arguments:
+            frames (np.ndarray): Frames on which predictions need
+            to be performed
+        Returns:
+            face boxes (List[List[BoundingBox]])
+        """
+
+        copy = torch.squeeze(frames)
+        transform = T.ToPILImage()
```
Comment (gaurav274) [inline eva/udfs/face_detector.py:67]: This converison is wierd. Are we doing it frame by frame?

Comment (jarulraj) [reply eva/udfs/face_detector.py:67]: Should we do it in some other manner?

```
        pil_image = T.ToPILImage(torch.squeeze(frames))
        bboxes, scores = self.model.detect(img=pil_image)
```

Comment (jarulraj) [reply eva/udfs/face_detector.py:67]: model takes in ndarray or PIL image

Comment (gaurav274) [reply eva/udfs/face_detector.py:67]: We need to provide a cleaner interface.

**Pull request merged**